### PR TITLE
COVID-19 Screener Content Updates/Style Tweaks

### PR DESCRIPTION
**Pull request by batemapf**
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by mchelen-gov — APPROVED** [review]

**Comment by mchelen-gov** [comment]
do we want to get the result page design updates in this PR or do it in a new one?

**Comment by batemapf** [comment]
this one is fine

File: src/applications/coronavirus-screener/components/FormResult.jsx [hunk 1]
```diff
@@ -5,13 +5,18 @@ import moment from 'moment';
...
 
   const incomplete = <div>Please answer all the questions above.</div>;
 
+  const dateText = moment().format('dddd, MMMM D, h:mm a');
+
   const pass = (
     <div>
-      <h2 className="vads-u-font-size--h1">PASS</h2>
-      <h3>Valid for: {moment().format('MMMM Do YYYY, h:mm a')}</h3>
+      <i class="fas fa-check"></i>
```
[the remark this answers is not in the document]
Comment (jenstrickland) [reply src/applications/coronavirus-screener/components/FormResult.jsx:16]: The icon markup should be:
`<i aria-hidden="true" role="presentation" class="fas fa-check"></i>`

Reason: in this usage, the heading it pairs with communicates the message 'OK to proceed' and it wouldn't add any additional context for non-sighted users. It is more of a graphic enhancement.

File: src/applications/coronavirus-screener/components/FormResult.jsx [hunk 2]
```diff
@@ -5,13 +5,18 @@ import moment from 'moment';
...
 
   const incomplete = <div>Please answer all the questions above.</div>;
 
+  const dateText = moment().format('dddd, MMMM D, h:mm a');
+
   const pass = (
     <div>
-      <h2 className="vads-u-font-size--h1">PASS</h2>
-      <h3>Valid for: {moment().format('MMMM Do YYYY, h:mm a')}</h3>
+      <i className="fas fa-check" />
```
[the remark this answers is not in the document]
Comment (jenstrickland) [reply src/applications/coronavirus-screener/components/FormResult.jsx:16]: The icon markup should be:
<i aria-hidden="true" role="presentation" class="fas fa-check"></i>

Reason: in this usage, the heading it pairs with communicates the message 'Pass' and it wouldn't add any additional context for non-sighted users. It is more of a graphic enhancement.

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**